addons/website_event/static/src/xml/event_create.xml

```diff
@@ -5,7 +5,7 @@
             <div class="form-group row" id="name">
                 <label for="name" class="col-md-2 col-form-label">Event Name</label>
                 <div class="col-md-10">
-                    <input type="text" name="name" class="form-control" required="required"/>
+                    <input type="text" name="name" class="form-control" required="required" placeholder="e.g. Conference for Architects"/>
```

Comment:
Linting: could be multi-lined as it creates long lines. We don't have hard rules for line length, we try to keep it somehow around max 100-120 (depending on what remains), aka
```
<input type="text" name="name" class="form-control" required="required"
       placeholder="e.g. Conference for Architects"/>
```